src/System.Management.Automation/engine/remoting/common/RunspaceConnectionInfo.cs

```diff
@@ -2217,7 +2217,35 @@ private static string[] ParseArgv(ProcessStartInfo psi)
         {
             var argvList = new List<string>();
             argvList.Add(psi.FileName);
-            argvList.AddRange(psi.Arguments.Split(' '));
+
+            var argsToParse = psi.Arguments.Trim();
+            var argsLength = argsToParse.Length;
+            for (int i=0; i<argsLength; )
+            {
+                var iStart = i;
+         
+                switch (argsToParse[i])
+                {
+                    case '"':
+                        // Special case for arguments within quotes
+                        // Just return argument value within the quotes
+                        while ((++i < argsLength) && argsToParse[i] != '"') { };
```

Comment:
No enforcement here.  This expects the argument string to be correctly formatted.  If it is not then the argument list is wrong and any errors are generated when the process is created.

This is currently scoped internally so we currently have complete control in how it is used.  However, I did intend this to be robust against malformed strings (empty, single quote, unmatched quotes, single space, only spaces, etc.).  If you see something I missed please let me know.